scripts/local-mode.sh

```diff
@@ -1,37 +1,63 @@
 #!/usr/bin/env bash
 
 function show_help() {
-    echo "Usage: $0 [clean|shell]"
+    echo "Usage: $0 [--clean|--shell]"
     echo "Setup/Starts a local InfluxDB container for Formant Local Mode"
     echo
     echo "Options:"
-    echo "  clean    Clean up the local InfluxDB container and remove data directory"
-    echo "  shell    Print the environment variables to be set in the agent.bashrc after the container is started"
+    echo "  --clean    Clean up the local InfluxDB container and remove data directory"
+    echo "  --shell    Print the environment variables to be set in the agent.bashrc after the container is started"
     echo
     echo "Environment Variables:"
     echo "  INFLUX_PORT              InfluxDB port (default: 8086)"
-    echo "  INFLUX_USERNAME          InfluxDB username"
-    echo "  INFLUX_PASSWORD          InfluxDB password"
+    echo "  INFLUX_USERNAME          InfluxDB username (default: admin)"
+    echo "  INFLUX_PASSWORD          InfluxDB password (default: admin123)"
```

Comment:
This is not safe. A better approach is to have the user provide a password (through CLI)

Reply:
We could instead do something like this to prompt the user to provide the password if not set
```
if [ -z "$INFLUX_USERNAME" ]; then
    read -p "Enter InfluxDB username: " INFLUX_USERNAME
fi

# Prompt for password if not set
if [ -z "$INFLUX_PASSWORD" ]; then
    read -s -p "Enter InfluxDB password: " INFLUX_PASSWORD
    echo
fi
```